functions/go/bind/pkg/meta/doc.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package meta
```

Comment:
this library name looks very familiar to me.

Reply:
Ah, https://github.com/GoogleContainerTools/kpt-functions-sdk/pull/627/files I assume this code will be changed to import the lib from the sdk

Reply:
Right - the goal here is to prove that these libraries are stable and worthy of inclusion in the library.  So if we can iterate on them here, that seems like a good way to do that!